### ISSUE TITLE
Terminal app v3.1.0 (Linux): missing ‘/info’ command

### ISSUE THREAD

**Issue by kseistrup** [simplex-chat/simplex-chat#907]
The release notes for v3.1.0 mentions:

* `/info` command to show information and servers for contacts and group members

However:

```
> /version
SimpleX Chat v3.1.0
To update run: curl -o- https://raw.githubusercontent.com/simplex-chat/simplex-chat/master/install.sh | bash
> /info
bad chat command: Failed reading: empty
```

This is the `simplex-chat-ubuntu-20_04-x86-64` binary running on ArchLinux.

**Comment by epoberezkin** [comment]
It’s `info <contact>` or `/info #<group> <member>`

**Comment by epoberezkin** [comment]
Will update the notes

**Comment by kseistrup** [comment]
Ah, so it's `/info $something`. Thanks!

**Comment by kseistrup** [comment]
It would be helpful if wrong usage showed the syntax of the command. As per your example:

```
> /info
usage: info <contact>
   or:  /info #<group> <member>
```

An error message of “bad chat command: Failed reading: empty” is unhelpful because it doesn't tell the user what went wrong or how to use the command properly, and even consulting the `/help command` is unhelpful in this case.

I suspect that at least some of the other commands suffer from the same unhelpfulness.

The proper usage _could_ be shown in the `/help` command, but by showing it explicitly after wrong usage helps the user immediately and saves them from reading through all the text from the `/help` command (some of which may require more than one invocation).

**Comment by epoberezkin** [comment]
Yeah, it would complicate parsing quite a bit - definitely possible but not a quick fix…

**Comment by epoberezkin** [comment]
Although maybe it’s just a case of making individual help for all commands and have the last catch all parser that grabs the first word as the command and maps it to the help…
